src/client/pfs/pfs.proto

```diff
@@ -509,14 +509,14 @@ message FileInfoNewStorage {
 //     Client sends a request with the EOF field set to true.
 //   Client connection is closed.
 
-message PutTarRequest {
+message PutTarRequestV2 {
```

Comment:
Yeah, my general approach has been to make all the relevant proto messages / rpcs V2 to be explicit. I think this is a sensible approach because there is really no downside and the upside is that we clearly distinguish the V2 messages / rpcs from the non-V2 messages / rpcs.